assets/js/atomic/blocks/product-elements/price/style.scss

```diff
@@ -1,11 +1,11 @@
 /*rtl:begin:ignore*/
-.wc-block-components-product-price__align-left {
+.wc-block-components-product-price--align-left {
 	text-align: left;
 }
-.wc-block-components-product-price__align-center {
+.wc-block-components-product-price--align-center {
 	text-align: center;
 }
-.wc-block-components-product-price__align-right {
+.wc-block-components-product-price--align-right {
 	text-align: right;
```

Comment:
this change is unrelated to the issue, right? just styling fixes?

Reply:
Yes, I wanted to align all class names to follow the same pattern (`--` instead of `__`) even in blocks where there was not an issue.